### DISCUSSION TITLE
vCannot get it to work at all

### DISCUSSION THREAD

**Discussion by zeberdee** [sultim-t/xash-rt#87] [Q&A]
I just cannot get this to work at all.

I;ve downloaded the 2 zip files, extracted them exactly as instructed, and when I try to run the Xash3d, I get a console appear, with the following errors.

![capture xash3d](https://user-images.githubusercontent.com/10728154/221554190-340f628c-84dc-4bf2-8740-6f158f158315.jpg)

and this is what i get when i run hl directly in the console

No detail texture mapping file: maps/c0a0_detail.txt
Unknown command: VModEnable
Unknown command: rt_classic_toggle

Can anybody tell me whats going wrong, and how I fix it.

My graphics card is an ATI Radion RX 6600 GT model

**Comment by palaashatri** [comment]
Can you try verifying your game files? I don't have an AMD GPU to test, but it ran flawlessly fine on my RTX 3050 laptop after just dragging and dropping in all the files + the DLSS DLL.

**Reply by zeberdee** [reply]
all files verified, still not working.
